src/init.lua

```diff
@@ -116,6 +116,36 @@ function osm2pgsql.make_clean_tags_func(keys)
     end
 end
 
+-- from http://lua-users.org/wiki/StringTrim
+function osm2pgsql.trim(str)
+    local from = str:match("^%s*()")
+    return from > #str and "" or str:match(".*%S", from)
+end
+
+function osm2pgsql.split_unit(str, default_unit)
+    local val, unit = string.match(str, "^([0-9.]+) ?(%a*)$")
```

Comment:
Do we want to allow negative values?

Reply:
I had thought about that and don't see where this would be useful. `width`, `maxspeed` can only be positive. Maybe for `ele`?

Reply:
Its a low-level function that shoulde decide what the correct range of values is, so I have changed it to allow negative values.